mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -1881,6 +1885,113 @@ def __call__(
         )
 
 
+@experimental(version="3.7.0")
+@format_docstring(_MODEL_API_DOC)
+class Summarization(BuiltInScorer):
+    """
+    Summarization evaluates whether a summarization output is factually correct, grounded in
+    the input, and does not make any assumptions not present in the input.
+
+    This scorer focuses on three key aspects:
```

Comment:
nit: let's not discuss implementation details in the docstring as this can change and we may forget to update it.